### DISCUSSION TITLE
How can I save cookie with multiple objects in array?

### DISCUSSION THREAD

**Discussion by Domino-98** [nuxt/nuxt#17358] [Questions]
I want to save my array of products from Pinia store in cookie but it sets the first pushed object only.
```
  const wishlist = useCookie("wishlist");
  wishlist.value = productStore.getWishlistProducts;
```
![image](https://user-images.githubusercontent.com/64014834/176522869-b5511f2a-66db-45bd-9d4b-900d7476dd8c.png)
Anyone knows how to fix it?

**Comment by Domino-98** [comment]
Nevermind. Local storage works.
```
      if (process.client) {
          localStorage.setItem("wishlist", JSON.stringify(this.wishlistProducts));
      }
```

**Comment by Flanker72** [comment]
Cookie isn't the right place to save this data. Cookies have 4096 bytes limit and support plain strings only. However the cookie with only one product from your example has 2510 bytes of length. Seems you cannot add more. Use localStorage for such data instead.
